main.py

```diff
@@ -128,8 +96,45 @@ def get_jenkins_jobs(server, job_search_fields):
 	return relevant_jobs
 
 
+def get_bugs_dict(bug_ids):
+	''' takes in set of bug_ids and returns dictionary with
+		bug_ids as keys and API data as values
+	'''
+
+	# initialize bug dictionary
+	bugs = {}
+
+	# iterate through bug ids from set
+	for bug_id in bug_ids:
+
+		# 0 should be default in YAML file (i.e. no bugs recorded)
+		# if present reference should be made in bug dict
+		if bug_id == 0:
+			bugs[0] = {'bug_name': 'No bug on file', 'bug_url': None}
+			continue
+
+		# get bug info from bugzilla API
+		try:
+
+			# hotfix: API call does not work if '/' present at end of URL string
+			parsed_bz_url = config['bz_url'].rstrip('/')
+
```

Comment:
Line break intentional?

Reply:
Yeah I have this line isolated since it is a hotfix and may need to be changed at a later date

Reply:
Got it